tests/test_image.py

```diff
@@ -67,14 +67,16 @@ def test_FindRegions(warn_empty, recwarn):
         assert num_yielded_objects > 0
 
 
-def test_ExtractROI():
+@pytest.mark.parametrize(
+    "alpha,bg_color",
```

Comment:
Other different combinations should also be checked. I would vary `alpha` (e.g. 0, 0.5, 1) and `bg_color` (e.g. "black", 0) independently. Also, `ExtractROI` must work with both gray and RGB images. In the latter case, `bg_color` could also be a triple (e.g. `(127,255,127)`). You could write two independent tests for gray and RGB.